website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -43,10 +43,12 @@ cheatcode-namespace.
 ### `strategy.felts`
 
 ```python
-def felts() -> Strategy: ...
+def felts(*, comparable: bool = False) -> Strategy: ...
 ```
 
-Explores all possible felt values.
+By default, explores all possible felt values.
+
+If `comparable` is true, explores felts which can be range checked, i.e. `assert_le` will not crash.
```

Comment:
```suggestion
If `comparable` is `True`, explores felts which can be range checked, e.g. `assert_le` will not crash.
```
1) Maybe it's worth to mention this is a keyword only argument.
2) Why does `assert_le` crashes in the first place? Shouldn't this option be enabled by default? I think this parameter needs more detailed description.

Reply:
1. Done
2. Not all felt values are allowed to be range checked, the limit is determined by the layout of the Cairo program (which as far as I understand, is the same for all "civil" Starknet contracts, hence the constant is hardcoded). Range checks are allowed only with felts from range `[0, RANGE_CHECK_BOUND)`. I think this comes from the fact that they are doing some bitwise ops in the builtin and these operations must not overflow. Whatever is default is a very philosophical question. I opted for maintaining compatibility with existing behaviour (`strategy.felts()` behaves the same as fuzzer behaved when it was released).

Reply:
IMO `comparable` is a misleading name. It could be understood felts provided by fuzzer can't be compared by default.